pkg/declarativeconfig/access_scope_test.go

```diff
@@ -9,31 +9,41 @@ import (
 )
 
 func TestAccessScopeYAMLTransformation(t *testing.T) {
-	data := []byte(`
-name: test-name
+	data := []byte(`name: test-name
 description: test-description
 rules:
-  included:
-    - cluster: clusterA
-      namespaces:
-      - namespaceA1
-      - namespaceA2
-    - cluster: clusterB
-  clusterLabelSelectors:
-    - requirements:
-      - key: a
-        operator: IN
-        values: [a, b, c]
-      - key: b
-        operator: NOT_IN
-        values: [b, d]
-    - requirements:
-      - key: c
-        operator: IN
-        values: [d, e, f]
-      - key: d
-        operator: NOT_IN
-        values: [x, y, z]
+    included:
+        - cluster: clusterA
+          namespaces:
+            - namespaceA1
+            - namespaceA2
+        - cluster: clusterB
+    clusterLabelSelectors:
+        - requirements:
+            - key: a
+              operator: IN
+              values:
+                - a
+                - b
+                - c
+            - key: b
+              operator: NOT_IN
+              values:
+                - b
+                - d
+        - requirements:
+            - key: c
+              operator: IN
+              values:
+                - d
+                - e
+                - f
+            - key: d
+              operator: NOT_IN
+              values:
+                - x
+                - "y"
+                - z
```

Comment:
Hmm, don't really see a reason for it? Its essentially interchangeable to have it either in this format or in the yaml list - the decoder will always provide the YAML list format, so let's keep it like this since its easier to compare.